Enterprise-CMCS/managed-care-review
# dev_tool: Remove request, replace with axios

## Summary
Remove depreacted dep `request` and replace it with `axios` 
#### Related issues
https://jiraent.cms.gov/browse/MCR-1792

#### Screenshots
n/a 

#### Test cases covered

n/a

## QA guidance

Please pull down this branch and run `./dev local` to check this work. I currently have a broken local dev for other reasons and can't fully check this work.
